numpy/core/src/multiarray/_multiarray_tests.c.src

```diff
@@ -161,10 +161,11 @@ test_neighborhood_iterator(PyObject* NPY_UNUSED(self), PyObject* args)
     PyArrayObject *ax, *afill;
     PyArrayIterObject *itx;
     int i, typenum, mode, st;
+    int idxstart = 0;
```

Comment:
For your information (this is just a test so, it doesn't matter), using `Py_Ssize_t` and then parsing with `n` is better.  NumPy uses `npy_intp` internally (which can differn unfortunately). But integers are 32bit on 64bit windows, so they are not quite right.

Reply:
Done